### PR TITLE
[IMP] mail : composer / chatter new followers email address tooltips.

### PR DESCRIPTION
**Pull request by mep-odoo**
Task : https://www.odoo.com/web?#id=36677&view_type=form&model=project.task&action=333&active_id=131&menu_id=4720

Pad : https://pad.odoo.com/p/r.e7f17baffe19636b821f5b6a3e0df41f

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by mba-odoo** [comment]
LGTM

File: addons/mail/static/src/js/many2many_tags_email.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ BasicModel.include({
      * @param {Object} record - an element from the localData
      * @param {string} fieldName
      * @return {Deferred<Object>} the deferred is resolved with the
-     *                            invalidPartnerIds and partnerNames
+     *                            invalidPartnerIds
      */
     _fetchSpecialMany2ManyTagsEmail: function (record, fieldName) {
```
Comment (alexkuhn) [inline addons/mail/static/src/js/many2many_tags_email.js]: This function doesn't "fetch" anymore.
Either rename to `_setSpecialMany2ManyTagsEmail`, or revert to `_setInvalidMany2ManyTagsEmail`.

File: addons/mail/static/src/js/many2many_tags_email.js [hunk 2]
```diff
@@ -43,26 +43,15 @@ BasicModel.include({
...
-                });
+            return $.when({
+                invalidPartnerIds: _.pluck(invalidPartnerIds, 'res_id'),
             });
         });
     },
 });
 
 var FieldMany2ManyTagsEmail = M2MTags.extend({
+    tag_template: "Many2ManyComposerTitle",
```
Comment (alexkuhn) [inline addons/mail/static/src/js/many2many_tags_email.js]: Template name is misleading. Maybe `FieldMany2ManyTagsEmail` is better?

File: addons/mail/static/src/js/many2many_tags_email.js (hunk 1 above)
Comment (mep-odoo) [reply addons/mail/static/src/js/many2many_tags_email.js]: @alexkuhn changes done.

**Pull request merged**